README.md

```diff
@@ -36,7 +36,45 @@ Need the copilot to comment, fix a bug, or add a doc string? Simply go to the co
 
 You can log in and out of your Pieces account using the `Pieces: Login` or `Pieces: Logout` command in your command palette. An output panel is available to check your cloud status and login status.
 
-### Connect with the Pieces Community
+### Connect/Disconnect your cloud
+
+You can connect and disconnect from your cloud using  `Pieces: Connect your cloud` or `Pieces: Disconnect your cloud` command in your command palette.(Login required!)
+
+### Search
+
+You can search a saved material using `Pieces: Search`.
+
+### Reload
+
+Run `Pieces: Reload the plugin` if you can't see any command in your command palette. Just make sure that **Pieces OS** is running
+
+### Support 
+
+Got stuck? No worries, Simply go to the command palette and run `Pieces: Get Support`. you will find some very awsome resources where you can connect with use
```

Comment:
"Experiencing an issue or have feedback for the Pieces team? No worries, we've got you covered. Simply open the command palette and run `Pieces: Get Support`. From here, you will find resources where you can connect with the Pieces team."